src/Abstractions/NexusMods.Abstractions.Installers/LoadoutDetails.cs

```diff
@@ -0,0 +1,22 @@
+using JetBrains.Annotations;
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Loadouts;
+
+namespace NexusMods.Abstractions.Installers;
+
+/// <summary>
+/// Provides details about a loaodut.
+/// </summary>
+[PublicAPI]
+public record LoadoutDetails
+{
+    /// <summary>
+    /// Gets the loadout.
+    /// </summary>
+    public required Loadout.ReadOnly Loadout { get; init; }
+
+    /// <summary>
+    /// Gets the game installation.
+    /// </summary>
+    public GameInstallation GameInstallation => Loadout.InstallationInstance;
```

Comment:
Is there a reason to pack this data together apart from passing one less parameter?

Reply:
Our current installers have `ModInstallerInfo` which contains a lot more properties. The new installer types don't need most of what's in there. I don't know yet what installers want in addition to this information, so I just made this new record for loadout related details.

Reply:
The installation is in the Loadout: `loadout.InstallationInstance`

Reply:
I can probably just pass `Loadout.ReadOnly` for now.